log/logger_test.go

```diff
@@ -20,9 +21,19 @@ func Test_Log(t *testing.T) {
 	a, buffer, log := Setup(t)
 
 	log.Log("my message")
-
-	a.Contains(buffer.String(), "my message")
-	a.Contains(buffer.String(), "level=info")
+	got := buffer.String()
+
+	a.Contains(got, "my message")
+	a.Contains(got, "level=info")
+
+	// check valid timestamp
+	idx := strings.Index(got, "time=")
```

Comment:
Minor nitpicky thing, but you may want to set a variable for `"time="` since you're using in two places here

Reply:
Good point, I originally had it as a var but duplicated it for readability.. but I changed it back in my latest commit